lib/org/openpsa/user/accounthelper.php

```diff
@@ -320,6 +328,27 @@ public function check_password_age() : bool
         return false;
     }
 
+    /**
+     * Function sends an email to the user with username and password
+     *
+     * @return boolean indicates success
+     */
+    public function welcome_email(midcom_db_person $person) : bool
+    {
+        $this->person = $person;
```

Comment:
do we really need to overwrite `person` here? It gets set in the constructor, no?

Reply:
removed